airflow/dags/disambiguation/inventor_assignee_disambiguation.py

```diff
@@ -18,7 +18,7 @@
     finalize_disambiguation, upload_results as upload_inventor_results
 from updater.disambiguation.location_disambiguation.location_disambiguator import *
 from updater.post_processing.post_process_location import post_process_location, post_process_qc
-from updater.post_processing.post_process_assignee import post_process_qc as qc_post_process_assignee, \
+from updater.post_processing.post_process_assignee import additional_post_processing_assignee, post_process_qc as qc_post_process_assignee,  \
```

Comment:
I see the additional post processing is set to run after canonical gets created. I suppose the canonical gets recreated within additional post processing step?

Reply:
yes precache and create_canonical_assignees are regenerated in that step